pkg/aws_config_server/list_roles.go

```diff
@@ -271,7 +271,7 @@ func processAWSErr(err error) error {
 		return err
 	}
 	if awsErr.Code() == errAWSAccessDenied {
-		logrus.WithError(err).Errorf("AWS error %s", awsErr)
+		logrus.Errorf("AWS error: %s", awsErr.Message())
```

Comment:
I wonder if we actually want to keep the .WithError and then also use your new fix for the awsErr.Message

Reply:
Hmm. If I include the .WithError, we get repetitive info, like:
`ERRO[0007]: <details> error: <same details>`